src/components/navigation-bar/navigation-bar.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import NavigationBar from './navigation-bar';
+import Link from '../link/link';
+import notes from './notes.md';
+
+const handleClick = (ev) => {
+  ev.target.preventDefault();
+  action('click');
+};
+
+storiesOf('Navigation Bar', module)
```

Comment:
In the current demo site:

![image](https://user-images.githubusercontent.com/26925206/53879233-da1f2880-4005-11e9-8f0f-16119a5ffe70.png)

In this implementation:

![image](https://user-images.githubusercontent.com/26925206/53879272-ee632580-4005-11e9-99b7-e653fa4428d2.png)

Reply:
If I leave the existing demo site setup in place, the component will be invisible initially (white Nav on a white background with no children). And having a text knob for `children` felt like an odd way to demo it, as not really representative of what would go in a nav bar, so I tried to go for something a little bit more conventional.

Happy to revert to what's on the current site, but it feels like re-styling what I've got here, or replacing the links with something else (a Menu maybe?) could be a better way to go?

Reply:
I think you may be able to wrap the component in a container, so that it is visible when first loaded. 

In general, though we want to replicate the same info that's on the demo site, first and foremost we want to display the components and their functionality. Feel free to get creative, even if it means a departure from the exact implementation on the Demo site.

Reply:
Fixed, and have added a minor box-shadow so component is visible at default settings